ha/setup/pre_disruptive_upgrade.py

```diff
@@ -102,18 +102,26 @@ def cluster_standby_mode() -> None:
         UpgradeError
     """
     Log.info("Set cluster to standby mode")
-    standby_cmd = "pcs node standby --all --wait=600"
+    Log.info("Please wait, standby can take max 20 to 30 min.")
+    standby_cmd = f"{PCS_CLUSTER_STANDBY} --wait=600"
```

Comment:
Is this wait time inline with above comment?

Reply:
yes, 600 sec mean 30 min 
so max wait time is 30 min

Reply:
updated